Tests/Controller/AzineEmailControllerTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+namespace Azine\EmailBundle\Tests\Controller;
+use Azine\EmailBundle\DependencyInjection\AzineEmailExtension;
+use Azine\EmailBundle\Services\AzineEmailTwigExtension;
+use Azine\EmailBundle\Services\Pagination;
+use Azine\EmailBundle\Tests\FindInFileUtil;
+use Azine\EmailBundle\Services\AzineTemplateProvider;
+use Azine\EmailBundle\Entity\SentEmail;
+use Azine\EmailBundle\Controller\AzineEmailTemplateController;
+use Azine\PlatformBundle\Services\EmailTemplateProvider;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\File\Exception\FileNotFoundException;
+use Symfony\Component\HttpFoundation\ParameterBag;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Storage\MockFileSessionStorage;
+use Symfony\Component\Routing\RequestContext;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Bundle\FrameworkBundle\Client;
+use Azine\EmailBundle\Tests\TestHelper;
+use Doctrine\ORM\EntityManager;
+
+class AzineEmailControllerTest extends WebTestCase
+{
+    public function testAdminEmailsDashboardAction()
+    {
+        $this->checkApplication();
+
+        // Create a new client to browse the application
+        $client = static::createClient();
+        $client->followRedirects();
+
+        $manager = $this->getEntityManager();
+        $sentEmailRep = $manager->getRepository("Azine\EmailBundle\Entity\SentEmail");
+
+        $testSentEmails = $sentEmailRep->search(['recipients' => TestHelper::TEST_EMAIL]);
+
+
+        if (count($testSentEmails) == 0) {
+
+            TestHelper::addSentEmails($manager);
```

Comment:
This adds one slingle `sent_email` that has been sent to `test@mail.com` so the filtering test will not prove anything. Right?! Or where are the other `sent_emails` added/coming from?
The db should contain multiple "pages" of emails with different recipients.